pysimplesoap/client.py

```diff
@@ -37,6 +37,7 @@
                      get_local_name, get_namespace_prefix, TYPE_MAP, urlsplit, \
                      iteritems
 from .wsse import UsernameToken
+from urllib3.util import make_headers
```

Comment:
In that case, I'll just add a "make_headers" function to the urllib3Transport class and leave the rest as is.
Thanks